perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -445,9 +447,46 @@ sub front_stats_data {
         updates => $updates,
         new     => $new,
         recency => $recency,
+        _generated => time(),
     };
```

Comment:
So I guess having a third function that does the actual stats generation would make that easier.